platformio.ini

```diff
@@ -25,28 +25,29 @@ lib_deps =
   Adafruit NeoPixel
   https://github.com/lincomatic/LiquidTWI2.git
   https://github.com/trinamic/TMC26XStepper.git
+build_flags = -I $BUILDSRC_DIR -fmax-errors=5
```

Comment:
Please add a comment here to clarify how to see all errors

Reply:
I've added one in the heading.

Reply:
Looks good to me